noether_gui/src/widgets/tpp_widget.cpp

```diff
@@ -81,18 +84,20 @@ TPPWidget::TPPWidget(boost_plugin_loader::PluginLoader loader, QWidget* parent)
   render_widget_->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
 
   // Set visibility of the actors based on the default state of the check boxes
-  mesh_actor_->SetVisibility(ui_->check_box_show_original_mesh->isChecked());
-  mesh_fragment_actor_->SetVisibility(ui_->check_box_show_modified_mesh->isChecked());
-  unmodified_tool_path_actor_->SetVisibility(ui_->check_box_show_original_tool_path->isChecked());
-  tool_path_actor_->SetVisibility(ui_->check_box_show_modified_tool_path->isChecked());
+  mesh_actor_->SetVisibility(ui_->action_showOriginalMesh->isChecked());
+  mesh_fragment_actor_->SetVisibility(ui_->action_showModifiedMesh->isChecked());
+  unmodified_tool_path_actor_->SetVisibility(ui_->action_showOriginalToolPath->isChecked());
+  tool_path_actor_->SetVisibility(ui_->action_showModifiedToolPath->isChecked());
 
   // Connect signals
-  connect(ui_->push_button_load_mesh, &QPushButton::clicked, this, &TPPWidget::onLoadMesh);
-  connect(ui_->check_box_show_original_mesh, &QCheckBox::clicked, this, &TPPWidget::onShowOriginalMesh);
-  connect(ui_->check_box_show_modified_mesh, &QCheckBox::clicked, this, &TPPWidget::onShowModifiedMesh);
-  connect(ui_->check_box_show_original_tool_path, &QCheckBox::clicked, this, &TPPWidget::onShowUnmodifiedToolPath);
-  connect(ui_->check_box_show_modified_tool_path, &QCheckBox::clicked, this, &TPPWidget::onShowModifiedToolPath);
-  connect(ui_->push_button_plan, &QPushButton::clicked, this, &TPPWidget::onPlan);
+  connect(ui_->action_loadMesh, &QAction::triggered, this, &TPPWidget::onLoadMesh);
+  connect(ui_->action_executePipeline, &QAction::triggered, this, &TPPWidget::onPlan);
+
+  connect(ui_->action_showOriginalMesh, &QAction::triggered, this, &TPPWidget::onShowOriginalMesh);
+  connect(ui_->action_showModifiedMesh, &QAction::triggered, this, &TPPWidget::onShowModifiedMesh);
+  connect(ui_->action_showOriginalToolPath, &QAction::triggered, this, &TPPWidget::onShowUnmodifiedToolPath);
+  connect(ui_->action_showModifiedToolPath, &QAction::triggered, this, &TPPWidget::onShowModifiedToolPath);
```

Comment:
Being a bit nitpicky, but I would prefer if the UI variable names were all `snake_case` with the type of UI object as the first word and the description after it (e.g., `action_ShowOriginalMesh` -> `action_show_original_mesh`)